Author: Jordi Mallach <jordi@debian.org>
Description: Show a warning if the alsa wrapper failed.
 If /proc/asound is not found, throw a warning and continue without
 any wrapping.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=404040
Forwarded: no

--- alsa-oss/alsa/aoss.in	2008-07-14 10:58:00.000000000 +0200
+++ alsa-oss/alsa/aoss.in	2010-02-12 02:04:19.000000000 +0100
@@ -9,6 +9,7 @@
   libdir=@libdir@
   LD_PRELOAD=${libdir}/libaoss.so${LD_PRELOAD:+:$LD_PRELOAD} exec "$@"
 else
+  echo "Warning: /proc/asound not found. Running without ALSA wrapper."
   exec "$@"
 fi
 exit 1
